app/components/Settings/AlphaDownload.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from 'react';
+import { Button, ListItem, makeStyles } from '@material-ui/core';
+import { GetApp as DownloadIcon } from '@material-ui/icons';
+
+const useStyles = makeStyles({
+  root: {
+    justifyContent: 'flex-end',
+  },
+});
+
+const AlphaDownload = () => {
+  const styles = useStyles();
+
+  const handleClick = useCallback(() => {
+    const win = window.open("https://web.archive.org/web/20150726160905if_/http://alexbeachum.com/outerwildsDownloads/OuterWilds_Alpha_1_2_PC.zip", 'Wayback Machine');
+    if (win != null) {
+      win.focus();
+    }
+  }, []);
+
+  return (
+    <ListItem className={styles.root}>
+      <Button
+        variant="contained"
+        onClick={handleClick}
+        startIcon={<DownloadIcon />}
+      >
+        {'Download Outer Wilds Alpha'}
+      </Button>
+    </ListItem>
+  );
+};
```

Comment:
Instead of a direct download button, let's just have a button that points to `https://outerwildsmods.com/outer-wilds-alpha/`. This way we don't have to change two places if we ever change how to download the alpha (unless we make it downloadable via the database like a mod), and we reduce the chance of getting into trouble. The button text should of course also change to reflect this.